app/views/users/index.html.erb

```diff
@@ -1 +1,15 @@
-<h1>List of Users</h1>
+<h1>Listing users</h1>
+<div class="container">
+  <h2>Users</h2>
+  <% @users.each do |user| %>
+    <%= link_to user_path(user) do %>
+      <div class="user-list">
+        <p><%= user.photo %></p>
```

Comment:
- [ ] As required in the project requirement, Kindly make sure you stick to the original wireframe that is provided in the project requirements. as you may notice the difference.

### Your template :point_down: 
![image](https://github.com/saluumaa/blog-app/assets/97100091/4197c3c8-6a27-4ed8-bc00-d61280c7921c)
### Original template :point_down: 
![Screenshot from 2023-07-21 15-18-39](https://github.com/saluumaa/blog-app/assets/97100091/5db6b100-4cbd-4a9c-9834-b62bbe44a6c4)
Hint: `To achieve that, you may consider,`
```ruby
#Grab your image inside an anchor img tag instead of a paragraph tag p as you just did in here.
<img src="<%= user.photo %> " >

#Also make sure you user the flexbox or grid container to display your contents to match the original design
```